Source/WebCore/platform/graphics/PathStream.h

```diff
@@ -83,12 +123,14 @@ class PathStream final : public PathImpl {
     std::optional<PathDataQuadCurve> singleQuadCurve() const final;
     std::optional<PathDataBezierCurve> singleBezierCurve() const final;
 
-    bool isEmpty() const final { return m_segments.isEmpty(); }
+    bool isEmpty() const final { return segments().isEmpty(); }
```

Comment:
Calling segments(), calls DataRef::access() - does that trigger a copy? From the commit message I would expect only non-const functions to be able to call segments()? I might confuse something...

Reply:
There's a `const` version of `segments()` higher up in the header, but it's exported so not inline. I'll change this one to access the data member directly.

Reply:
Ah it wouldn't even have compiled without the presence of a non-const accessor, missed that. Thanks Simon!